crates/runtime-object-store/Cargo.toml

```diff
@@ -31,7 +34,10 @@ tokio.workspace = true
 tracing.workspace = true
 url.workspace = true
 
+[dev-dependencies]
+insta = { workspace = true, features = ["json", "redactions"] }
+
 [features]
-ftp = ["dep:suppaftp", "dep:ssh2"]
-nfs = ["dep:libnfs"]
-smb = ["dep:smb"]
+ftp = ["dep:suppaftp", "dep:ssh2", "dep:bb8"]
+nfs = ["dep:libnfs", "dep:nix"]
```

Comment:
This will break the Windows build - we should only enable this for non-Windows builds